client-cli/src/test/java/org/eclipse/dataspaceconnector/registration/cli/ParticipantsCommandTest.java

```diff
@@ -47,7 +47,8 @@ class ParticipantsCommandTest {
     Participant participant2 = createParticipant();
     String serverUrl = FAKER.internet().url();
     String idsUrl = FAKER.internet().url();
-    String did = FAKER.internet().url();
+    String clientDid = FAKER.internet().url();
+    String dataspaceDid = "did:web:" + FAKER.internet().url();
```

Comment:
nit:
```suggestion
    String dataspaceDid = "did:web:" + FAKER.internet().domainName();
```

because with `FAKER.internet().url()` this becomes - www.domain but i think it should be only domain.